resources/install.rb

```diff
@@ -21,6 +21,7 @@
 
 property :source, String
 property :additional_components, Array, default: []
+property :install_method, [:windows_feature_dism, :windows_feature_powershell, :windows_feature_servermanagercmd], required: false
```

Comment:
You need to set a property type and then allowed symbols here, I think this is wrong. 

Best to look up Property types, https://docs.chef.io/custom_resources/#property (Which should be a symbol in this case) and validators to ensure it is equal to one of the allowed symbols you have set above: https://docs.chef.io/custom_resources/#validators

I would also set a default here so that the code does not need to change on line 34